src/android/UploadTask.java

```diff
@@ -357,6 +357,7 @@ private Request createRequest() throws FileNotFoundException {
         }
 
         bodyBuilder.addFormDataPart(fileKey, filepath, fileRequestBody);
+        bodyBuilder.setType(MultipartBody.FORM);
```

Comment:
@azharbeebeejaun Tested locally and it works. Can we merge?

In iOS we also use Multipart Form Request:

https://github.com/spoonconsulting/cordova-plugin-background-upload/blob/dfa0f4267b9a08b849c64b4ad0b76c1fd0191b39/src/ios/FileUploader.m#L131